etc/security/mh_default_org.xml

```diff
@@ -507,6 +507,56 @@
   </bean>
   -->
 
+  <!-- ################ -->
+  <!-- # LDAP Support # -->
+  <!-- ################ -->
+
+  <!--
+  <bean id="contextSource"
+    class="org.springframework.security.ldap.DefaultSpringSecurityContextSource">
+    < ! - - URL of the LDAP server - - >
+    <constructor-arg value="ldap://myldapserver:myport" />
+    < ! - - "Distinguished name" for the unprivileged user - - >
+    < ! - - This user is merely to perform searches in the LDAP to find the users to login - - >
+    <property name="userDn" value="uid=user-id,dc=example,dc=com" />
+    < ! - - Password of the user above - - >
+    <property name="password" value="mypassword" />
+  </bean>
+  -->
+
+  <!--
+  <bean id="ldapAuthProvider"
+    class="org.springframework.security.ldap.authentication.LdapAuthenticationProvider">
+    <constructor-arg>
+      <bean
+        class="org.springframework.security.ldap.authentication.BindAuthenticator">
+        <constructor-arg ref="contextSource" />
+        <property name="userDnPatterns">
+          <list>
+            < ! - - Dn patterns to search for valid users. Multiple "<value>" tags are allowed - - >
+            <value>uid={0},dc=example,dc=com</value>
+          </list>
+        </property>
+        < ! - - If your user IDs are not part of the user Dn's, you can use a search filter to find them - - >
+        < ! - - This property can be used together with the "userDnPatterns" above - - >
+        < ! - -
+        <property name="userSearch">
+          <bean name="filterUserSearch" class="org.springframework.security.ldap.search.FilterBasedLdapUserSearch">
+            < ! - - Base Dn from where the users will be searched for - - >
+            <constructor-arg index="0" value="ou=GroupName,dc=my-institution,dc=country" />
+            < ! - - Filter to located valid users. Use {0} as a placeholder for the login name - - >
+            <constructor-arg index="1" value="(uid={0})" />
+            <constructor-arg ref="contextSource" />
+          </bean>
+        </property>
+        - - >
+      </bean>
+    </constructor-arg>
+    < ! - - Defines how the user attributes are converted to authorities (roles) - - >
```

Comment:
@lkiesow Please check my point with the comments, what do you think? Besides that detail is ok for me to merge.

Reply:
Unfortunately, we cannot just change that since you cannot use an XML comment within an XML comment. The notation of adding spaces – which need to be removed when removing the outer comment – is what the previous LDAP configuration was already using and I did not have a better idea than that:

https://github.com/opencast/opencast/blob/71480e193b927ecf4397bdeffc1fecd99f625e86/etc/security/security_sample_ldap.xml-example#L519-L522